usage-limits.md

```diff
@@ -0,0 +1,54 @@
+---
+
+copyright:
+  years: 2015, 2018
+lastupdated: "2018-03-08"
+
+---
+
+{:shortdesc: .shortdesc}
+{:new_window: target="_blank"}
+{:tip: .tip}
+{:pre: .pre}
+{:codeblock: .codeblock}
+{:screen: .screen}
+{:javascript: .ph data-hd-programlang='javascript'}
+{:java: .ph data-hd-programlang='java'}
+{:python: .ph data-hd-programlang='python'}
+{:swift: .ph data-hd-programlang='swift'}
+
+# Usage limits
+
+The following usage limits and restrictions apply to {{site.data.keyword.nlushort}} service instances.
+{: shortdesc}
+
+## Analyzed text limit
+
+{{site.data.keyword.nlushort}} truncates analyzed text that contains more than 50,000 single-byte or multibyte characters. To view the text that counts toward this limit in your requests, set the `return_analyzed_text` parameter to `true`.
+
+You can set a smaller character limit with the `limit_text_characters` parameter. If you are interested in analyzing only a small portion of your content, this can help you avoid excessive costs.
+
+Example parameters object:
+```json
+{
+  "url": "ibm.com",
+  "limit_text_characters": 10000,
+  "return_analyzed_text": true,
+  "features": {
+    "concepts": {}
+  }
+}
+```
+
+## Concurrent request limit
+
+Each {{site.data.keyword.nlushort}} service instance is limited to 30 concurrent requests. It's possible that this limit may decrease for service instances on the Lite and Standard plans when the system is experiencing exceptionally heavy traffic. For example, an application that makes 35 simultaneous requests from a single service instance will exceed the concurrent request limit by five requests, and five of those requests will return a `429: Too Many Requests` error.
+
+To increase your concurrent request limit, [open a support ticket](https://ibm.biz/ibmcloudsupport).
```

Comment:
Do we really want to give them an explicit way to do this? Perhaps a contact sales if you need more than the standard service plans offer? Support seems like it's "something is wrong" issue, where it's really a "you're not paying enough to tax the system this much" issue.

Reply:
Ran this by OM, initially I had "upgrade to Premium" but they can work with you even on the Standard plan to increase the limit if you open a support ticket.